.circleci/config.yml

```diff
@@ -809,3 +811,42 @@ jobs:
             docker load < docker-image.tar
             docker tag quay.io/influxdb/oss-acceptance:${CIRCLE_SHA1} quay.io/influxdb/oss-acceptance:latest
             docker push quay.io/influxdb/oss-acceptance:latest
+  changelog:
+    machine:
+      image: ubuntu-2004:202010-01
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - last-commit
```

Comment:
Is it beneficial to cache this file since the repo gets checked out anyway? I'm assuming the latest commit file is available after you checkout the repo at `scripts/ci/changelog-commit.txt`.

If we want to cache, I'm wondering about having a non-unique cache key like this. It seems like it should work as long as every time the `run` step is executed the `save_cache` is executed as well, but weird and unpredictable stuff seems to always find a way to happen.

Reply:
oh this is true. caching not necessary, thanks!